api/db/changelog/db.changelog-145-add-access-tier-to-cdr-version.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="thibault" id="changelog-145-add-access-tier-to-cdr-version">
+    <addColumn tableName="cdr_version">
+      <column name="access_tier" type="VARCHAR(40)"/>
```

Comment:
Is this an enum, or are these actually string IDs now? If it's the former, we should use a different approach here.

Reply:
Strings, referencing the env conf JSON.

The real version probably won't be a string.  Maybe an FK to a tier table ID.